src/actions/guides/database/querying.cr

```diff
@@ -469,7 +485,7 @@ class Guides::Database::Querying < GuideAction
       def recently_completed_admin_tasks
         admin(true)
           .join_tasks
-          .tasks { |task_query|
+          .where_tasks { |task_query|
```

Comment:
You'll need to pass a query object  now. `TaskQuery.new.completed(true).updated_at.gte(1.day.ago)`